nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -65,6 +65,8 @@
     HAVE_APEX = True
```

Comment:
We need to import `AttnMaskType` here if I read correctly:
```suggestion
    from apex.transformer.enums import AttnMaskType

    HAVE_APEX = True
```

Reply:
Thanks! Good catch